.circleci/config.yml

```diff
@@ -579,6 +586,53 @@ jobs:
           name: Build SPM Core integration test
           command: cd Tests/SPMTest/CoreSPMTest && xcodebuild -scheme CoreSPMTest -destination "platform=iOS Simulator,OS=15.0,name=iPhone 13 Pro Max" build
       
+  generate-docs-job:
+    parameters:
+      xcode:
+        type: string
+        default: "12.4.0"
+    macos:
+      xcode: << parameters.xcode >>
+    environment:
+      HOMEBREW_NO_AUTO_UPDATE: 1
+    steps:
+      - checkout
+      - *install-cocoapods-installation
+      - *prepare-netrc-file
+      - *install-mbx-ci
+      - *restore-cache-podmaster
+      - *restore-cache-cocoapods-installation
+      - run:
+          name: Generate docs
+          command: |
+            OUTPUT=$( echo << pipeline.git.tag >> | sed 's/^v//' )
+            OUTPUT="${OUTPUT:-documentation}"
+            echo "export OUTPUT=$OUTPUT" >> $BASH_ENV
+            OUTPUT="${OUTPUT}" scripts/document.sh 2>&1 | tee docs.output && ! egrep -e "(WARNING)|(USR)" docs.output
+      - *save-cache-podmaster
+      - *save-cache-cocoapods-installation
+      - when:
+          condition:
+            matches:
+              pattern: "^v.*\\..*\\..*$"
+              value: << pipeline.git.tag >>
+          steps:
+            - run:
+                name: Push docs to publisher-production
+                command: |
+                  git remote set-url origin https://x-access-token:$(mbx-ci github writer token)@github.com/mapbox/mapbox-navigation-ios.git
+                  git config user.email "release-bot@mapbox.com"
+                  git config user.name "Release SDK bot"
```

Comment:
Should this be something like “Mapbox” or “MapboxCI” for consistency with other release bots around Mapbox? Or just “Mapbox Releases”, since the e-mail address doesn’t seem to be specific to a particular SDK?

Reply:
Changed it to Mapbox Releases.